sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_v1/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+friendly_name: Aggregation for Active Users
+description: |-
+  Active users aggregated by Mozilla product and business dimensions
+  (attribution details, channel, country, city, date, device model,
+  distribution_id, os details, segment)
+  Proposal:
+  https://docs.google.com/document/d/1qvWO49Lr_Z_WErh3I3058A3B1YuiuURx19K3aTdmejM/edit?usp=sharing
+owners:
+- lvargas@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name: bqetl_analytics_aggregations
+  task_name: active_users_aggregates_v1
+bigquery:
+  time_partitioning:
+    field: submission_date
+    type: day
+    require_partition_filter: true
+    expiration_days: null
+  clustering:
+    fields:
+    - country
+    - app_name
+    - attribution_medium
+    - channel
+references: {}
```

Comment:
This should get automatically generated by CI using `bqetl dependency record` but apparently only when `references` isn't in `metadata.yaml` instead of when it's not there or it's empty. https://github.com/mozilla/bigquery-etl/blob/0235d6087cb1757e12326a83bfef1634dc6b1367/bigquery_etl/dependency.py#L198-L201
```suggestion
```
(though I'm not sure if `references` actually does anything)